Issues with compiling with gcc-7.3

```
/usr/bin/ld: x86_methods.o: relocation R_X86_64_32S against `.text' can not be used when making a PIE object; recompile with -fPIC
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
Makefile:70: recipe for target 'uarch-bench' failed
make: *** [uarch-bench] Error 1
```

So after googling a bit found that passing `-no-pie` as LD_FLAGS made it work.

More information:
https://wiki.ubuntu.com/SecurityTeam/PIE

--- a/Makefile
+++ b/Makefile
@@ -29,6 +29,7 @@ CFLAGS := $(COMMON_FLAGS)
 PFC_SRC := libpfc-timer.cpp libpfm4-support.cpp
 SRC_FILES := $(wildcard *.cpp) $(wildcard *.c) nasm-utils/nasm-utils-helper.c
 SRC_FILES := $(filter-out $(PFC_SRC), $(SRC_FILES))
+LDFLAGS += -no-pie
 
 ifeq ($(USE_LIBPFC),1)
 LDFLAGS += -Llibpfc '-Wl,-rpath=$$ORIGIN/libpfc/' -L$(PFM_LIBDIR) '-Wl,-rpath=$$ORIGIN/$(PFM_LIBDIR)/'

